### DISCUSSION TITLE
control global strain

### DISCUSSION THREAD

**Discussion by LiWen-HUST** [idaholab/moose#16604] [Q&A Modules: Phase field] [answered]
Hello all,
I am using Phase Field + Tensor Mechanics to simulate phase transformation. In my model, there is local elastic strain and the global strain should be zero. How can I control the global strain to be zero?
Thank you very much

Best regards
Li

**Comment by ngrilli** [comment, marked as answer]
@LiWen-HUST 

Dear Li Wen,

your question is not very clear.

I believe you mean that you want zero macroscopic strain in a representative volume.
To achieve that, you can set zero displacement boundary conditions on all boundaries.

Best Regards,
Nicolò Grilli
National University of Singapore

**Reply by LiWen-HUST** [reply]
That's my question. Thank you very much!
